pkg/openshift-sdn/proxy.go

```diff
@@ -109,9 +109,15 @@ func (sdn *OpenShiftSDN) runProxy(waitChan chan<- bool) {
 			klog.Fatalf("Unable to read IPTablesMasqueradeBit from config")
 		}
 
-		localDetector, err := proxyutiliptables.NewDetectLocalByCIDR(sdn.ProxyConfig.ClusterCIDR, iptInterface)
-		if err != nil {
-			klog.Fatalf("Unable to configure local traffic detector: %v", err)
+		var localDetector proxyutiliptables.LocalTrafficDetector
+		if sdn.ProxyConfig.ClusterCIDR == "" {
+			klog.Warningf("detect-local-mode set to ClusterCIDR, but no cluster CIDR defined")
```

Comment:
need to change this to something that makes sense like multiple cidrs not supported, blah...